config/config_test.go

```diff
@@ -167,13 +167,13 @@ func TestConfigWithLogDeclarations(t *testing.T) {
 	// log configure for builtin fake motors would apply for a log level of `warn`. This "overlayed"
 	// log level is not applied at config parsing time.
 	test.That(t, cfg.Components[2].Name, test.ShouldEqual, "right_motor")
-	test.That(t, cfg.Components[2].LogConfiguration.Level, test.ShouldEqual, logging.INFO)
+	test.That(t, cfg.Components[2].LogConfiguration, test.ShouldBeNil)
```

Comment:
`LogConfiguration` is now an "optional" field (represented by a pointer). We used to use `logging.INFO` as the default level, but it is semantically better to use `nil` to represent _no_ specified configuration.